clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -65,6 +68,12 @@ object ElasticsearchIndex extends Elastic4sAutoDerivation {
   val WgsCram: ElasticsearchIndex[DocumentWgsCram] =
     indexDocument[DocumentWgsCram](version = 2)
 
+  /**
+    * Name to assign to a nested keyword field under every text field,
+    * to support exact matching.
+    */
+  val TextExactMatchFieldName: String = "exact"
```

Comment:
Not sure why this is defined here when it's only used in another class. Does this class have another attribute which forces this field to have this name? That would be a reason to keep it in here.